sources/packages/web/src/types/formio/custom-components/sims-components.ts

```diff
@@ -12,5 +12,6 @@ export const FORMIO_CUSTOM_COMPONENTS = {
     HEADER_MEDIUM,
     HEADER_SMALL,
     APPLICATION_EXCEPTION,
+    CURRENCY,
```

Comment:
The currency is missing to be imported. 

![image](https://github.com/user-attachments/assets/9cbfb5b4-0565-4836-9eb5-7023f2b55253)